src/normalize-keyframes.js

```diff
@@ -297,6 +301,7 @@
     return keyframes;
   }
 
+  shared.easingFunctionSymbol = window.Symbol ? Symbol('easingFunction') : '_easingFunction';
```

Comment:
As discussed offline, given that nothing else in the polyfill uses window.Symbol in this way, and it is not always available, I would rather just use _easingFunction throughout.